commands/failover.md

```diff
@@ -19,21 +20,22 @@ If the previous master had additional replicas attached to it, they will continu
 ## Optional arguments
 The following optional arguments exist to modify the behavior of the failover flow:
 
-`TIMEOUT` *milliseconds* -- This option allows specifying a maximum time a master will wait in the `waiting-for-sync` state before aborting the failover attempt and rolling back.
-This is intended to set an upper bound on the write outage the redis cluster can experience.
-Failovers typically happen in less than a second, but could take longer if there is a large amount of write traffic or the replica is already behind in consuming the replication stream. 
-If this value is not specified, the timeout can be considered to be "infinite".
+* `TIMEOUT` *milliseconds* -- This option allows specifying a maximum time a master will wait in the `waiting-for-sync` state before aborting the failover attempt and rolling back.
+    This is intended to set an upper bound on the write outage the Redis cluster can experience.
```

Comment:
What is this indentation for?  AFAIK it's not needed

Reply:
It was in Itamar's suggested edit's, I presume for readability? I was also under the impression just stripped out leading spaces, but, I don't know if we do anything fancy during rendering.

I didn't change any content, just typos and formatting.

Reply:
Not needed indeed, my fat finger on the tab or something

Reply:
The indention is needed so these lines are a part of the bullet.

Reply:
Sorry, not needed. Pushing a fix for that and also for a missing newline above.

Reply:
Ack.